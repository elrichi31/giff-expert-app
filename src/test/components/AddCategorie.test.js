const { shallow } = require("enzyme")
const { AddCategorie } = require("../../components/AddCategorie")
import React from 'react'

describe('Pruebas en el componente', () => {
    const setCategorie = () => {}
    const wrapper = shallow(<AddCategorie setCategories={setCategorie}/>)

    test('debe mostrarse correctamente', () => {
        expect(wrapper).toMatchSnapshot()
    })
    test('debe cambiar la caja de texto', () => {
        const input = wrapper.find('input')
        const value = 'hola'
        input.simulate('change', {target: {value}})
        expect( wrapper.find('p').text().trim()).toBe(value)
    })
    
    
})
