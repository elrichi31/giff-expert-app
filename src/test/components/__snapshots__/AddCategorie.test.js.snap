// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente debe mostrarse correctamente 1`] = `
<form
  className="form-group"
  onSubmit={[Function]}
>
  <input
    className="form-control"
    onChange={[Function]}
    type="text"
    value=""
  />
  <button
    className="btn btn-primary"
  >
    Search
  </button>
</form>
`;
